packages/flutter_tools/templates/create/.gitignore.tmpl

```diff
@@ -1,11 +1,12 @@
 .DS_Store
-.atom/
 .dart_tool/
-.idea
-.vscode/
+
 .packages
 .pub/
+
 build/
 ios/.generated/
-packages
+ios/Flutter/Generated.xcconfig
```

Comment:
these are already in templates/create/ios.tmpl/.gitignore

Reply:
Thanks, I wasn't aware of that.